Dockerfile

```diff
@@ -64,11 +64,12 @@ RUN echo "no" | avdmanager --verbose create avd --force --name "${EMULATOR_NAME}
 RUN curl -sL https://deb.nodesource.com/setup_18.x | bash && \
     apt-get -qqy install nodejs && \
     npm install -g npm && \
+    apt-get install cpu-checker && \
     npm i -g appium@next --unsafe-perm=true --allow-root && \
     appium driver install uiautomator2 && \
     exit 0 && \
     npm cache clean && \
-    apt-get remove --purge -y npm && \  
+    apt-get remove --purge -y npm && \
```

Comment:
Good one if we want to explicitly remove this package, 
but  running "apt-get autoremove --purge -y" should  remove the npm package as it was installed as dependency and no other package in need for it,  no ?